app/views/ChooseProvider.js

```diff
@@ -335,6 +321,19 @@ class ChooseProviderScreen extends Component {
             </TouchableOpacity>
           </MenuTrigger>
           <MenuOptions>
+            <TouchableOpacity
+              style={styles.authorityFilter}
+              onPress={() => this.toggleFilterAuthoritesByGPSHistory()}>
+              <Typography style={styles.authorityFilterText}>
```

Comment:
Try to use an appropriate use="body1" or 2 here if it fits.  And remove inline styles if so.

Reply:
Updated it for this element and the other 7 `<Typography>` elements in the file.